CrewChiefV4/R3E/R3EGameStateMapper.cs

```diff
@@ -917,9 +910,10 @@ public override GameStateData mapToGameStateData(Object memoryMappedFileStruct,
                                 }
                                 if (CarData.IsCarClassEqual(currentOpponentData.CarClass, currentGameState.carClass))
                                 {
+                                    float playerTimeByTyre = -1.0f;
```

Comment:
rename to playerClassSessionBestByTyre, as this better reflects what this variable contains - just in case we decide to use it again inside this block